### ISSUE TITLE
aws-auth ConfigMap formatting issue

### ISSUE THREAD

**Issue by uyenle57** [terraform-aws-modules/terraform-aws-eks#579]
# I have issues

## I'm submitting a...

* [*] bug report
* [ ] feature request
* [ ] support request - read the [FAQ](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/faq.md) first!
* [ ] kudos, thank you, warm fuzzy

## What is the current behavior?
Nodes were not able to connect to the cluster. After some debugging we found out that was because the `aws-auth` ConfigMap was not created correctly due to 2 empty lines in the `mapRoles` data pipe. Like so:

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: aws-auth
  namespace: kube-system
data:
  mapRoles: |
    - rolearn: arn:aws:iam::123455:role/cluster-name
      username: system:node:{{EC2PrivateDNSName}}
      groups:
        - system:bootstrappers
        - system:nodes


    - "groups":
      - "system:masters"
      "rolearn": "arn:aws:iam::123455:role/role-name"
      "username": "KubernetesAdmin"
```

Hence the ConfigMap was not applied and nodes weren't able to join the cluster.

## If this is a bug, how to reproduce? Please include a code sample if relevant.
Provide `map_roles` value e.g.
```
  map_roles = [
    {
      rolearn  = <ROLE-ARN>
      username = "KubernetesAdmin"
      groups   = ["system:masters"]
    },
  ]
```

## Are you able to fix this problem and submit a PR? Link here if you have already.
No.

## Environment details

* Affected module version: 6.0.2
* OS: Mac
* Terraform version: 0.12.8

## Any other relevant info

* EKS version: 1.13

**Comment by tmxak** [comment]
Why this issue was closed without any comments? I have the same issue with the latest module version

**Comment by uyenle57** [comment]
Hi @tmxak this is closed because it's a duplicate of #494. Please post your problem there instead.
